### DISCUSSION TITLE
[Dialog] Button control in the action slot

### DISCUSSION THREAD

**Discussion by cefiro** [tusen-ai/naive-ui#2148] [Q&A] [answered]
Hi!

I don't understand how to control the button in the dialog component.
I need to insert a button and manage its properties. Properties in vnode are not reactive.

How do I change the "disabled" property dynamically?
Example (Line 25): <https://codesandbox.io/s/lucid-bird-uidvi?file=/src/Demo.vue>

**Comment by 07akioni** [comment, marked as answer]
https://codesandbox.io/s/unruffled-dewdney-k09vt

**Reply by cefiro** [reply]
Thank you very much! You are the best!
